raiden-dapp/src/App.vue

```diff
@@ -79,7 +79,7 @@ export default class App extends Mixins(NavigationMixin) {
 }
 
 .v-main {
-  height: calc(100% - 120px);
+  height: calc(100% - 80px);
```

Comment:
The 80px is equivalent to the height of the app header, it used to be 120px but when I removed the address bar it is now only 80px. From what I know there is no way I can share this value between the style of the app header and this app component.

Reply:
Okay so the `App.vue` component is responsible to make the layout so that there is no overflow. This means you can't let the components inside decide that. You have to given them restrictions. Now you have an awkward mix. The size of the header  is defined within the header component. But the size of the body is in the App, depending on the header. That is the worst of all worlds. Please define the size for the header inside the App and calculate the height of the body depending on that.

Reply:
Improvements referenced in a new issue.